File: EEntityCore.DB/EEntityCore.DB.MSSQL/Interfaces/ISQLServerDBPointer.vb
Namespace Interfaces

    ''' <summary>
    ''' Mainly for pointing to a Specific Server Instance
    ''' </summary>
    ''' <remarks></remarks>
    Public Interface ISQLServerDBPointer

        ''' <summary>
        ''' Gets a new instance of SQL Server Connection
        ''' </summary>
        ''' <returns></returns>
        Function GetDBConn() As Server


    End Interface


End Namespace

File: EEntityCore.DB/EEntityCore.DB.MSSQL/Interfaces/ISQLServerDBPointer.cs

namespace EEntityCore.DB.MSSQL.Interfaces
{

    /// <summary>
    /// Mainly for pointing to a Specific Server Instance
    /// </summary>
    /// <remarks></remarks>
    public partial interface ISQLServerDBPointer
    {

        /// <summary>
        /// Gets a new instance of SQL Server Connection
        /// </summary>
        /// <returns></returns>
        Server GetDBConn();
    }
}